scm-ui/ui-components/src/Autocomplete.tsx

```diff
@@ -104,6 +105,7 @@ class Autocomplete extends React.Component<Props, State> {
                   },
                 });
               }}
+              aria-label={helpText || label}
```

Comment:
I cannot find any difference when this is there

Reply:
I had an example where the help-text was missing and therefore nothing was read out loud. So i thought it would be better to at least read the label in this case.